t/Geometry.t

```diff
@@ -21,6 +21,16 @@ ddg_spice_test(
         '/js/spice/geometry/',
         call_type => 'self',
         caller => 'DDG::Spice::Geometry'
+    ),
+    'area of circle' => test_spice(
+        '/js/spice/geometry/',
+        call_type => 'self',
+        caller => 'DDG::Spice::Geometry'
+    ),
+    'perimeter of square' => test_spice(
+        '/js/spice/geometry/',
+        call_type => 'self',
+        caller => 'DDG::Spice::Geometry'
```

Comment:
Can we get a bunch of tests for things that shouldn't trigger this? Like `area of a cone` etc...

Would be nice to have some more valid tests as well.